lte/gateway/c/session_manager/ChargingGrant.cpp

```diff
@@ -211,6 +223,13 @@ void ChargingGrant::log_final_action_info() const {
       final_action +=
           final_action_info.redirect_server.redirect_server_address();
     }
+    else if (final_action_info.final_action ==
+        ChargingCredit_FinalAction_RESTRICT_ACCESS) {
+      final_action += ", restrict_rules: ";
+      for (auto rule : final_action_info.restrict_rules) {
+        final_action += rule + ", ";
+      }
```

Comment:
A bit of a nit here, could we just do " " instead of ", "? The trailing comma is a bit of a pet peeve of mine... 
```
final_action += ", restrict_rules: {";
      for (auto rule : final_action_info.restrict_rules) {
        final_action += rule + "  ";
      }
final_action += "}"
```
?